src/styles/styles.ts

```diff
@@ -3518,12 +3541,15 @@ const styles = (theme: ThemeColors) =>
         },
 
         headerEnvBadge: {
-            marginLeft: 0,
-            marginBottom: 2,
+            position: 'absolute',
+            bottom: -8,
+            left: -8,
             height: 12,
+            width: 22,
             paddingLeft: 4,
             paddingRight: 4,
             alignItems: 'center',
+            zIndex: -1,
```

Comment:
Made sense to move the env Badge down, otherwise it would look weird to the right of the Expensify SVG
![image](https://github.com/Expensify/App/assets/85628901/770598ef-af1b-4080-90a4-4abbe07186e3)